### PR TITLE
add highlight class to button

### PR DESCRIPTION
**Pull request by tomltd**

### CODE REVIEW

File: src/js/scripts.js [hunk 1]
```diff
@@ -805,6 +805,10 @@ $('body').on('change', '[data-action="save-basic-info"]', function() {
 
 $('body').on('change', '[type="radio"]', function() {
 
+  //add highlight class to pick button
+  $('.loaded button.nav-link[data-action="pick"]').addClass( 'highlighted' );
+  console.log('button pressed');
```
Comment (katjam) [inline src/js/scripts.js]: oops!

**Comment by katjam** [comment]
:wheelchair:

**Comment by tomltd** [comment]
Classic, I even had a not on my monitor to remind me to take it out :-1:

**Comment by katjam** [comment]
so easily done.

**Pull request merged**